indigo_optional_drivers/ccd_andor/Makefile

```diff
@@ -42,5 +41,8 @@ $(BUILD_DRIVERS)/$(DRIVER_NAME): $(DRIVER_NAME)_main.o $(BUILD_DRIVERS)/$(DRIVER
 $(BUILD_DRIVERS)/$(DRIVER_NAME).$(SOEXT): $(DRIVER_NAME).o
 	$(CC) -shared -o $@ $^ $(LDFLAGS) -lindigo -landor
 
+install:
```

Comment:
Why do you need install target? The driver when built is placed in the same directory with other indigo drivers and will be installed from the main Makefile.

Reply:
I added the `make install` because running it on the main Makefile 
```console
# cd indigo/ 
# make
# cd indigo_optional_drivers/ccd_andor/
# make
# cd ../../
# make install # now back at indigo/ main folder
```

then, run:

```
[root@5dac5c4dbb17 indigo]# indigo_server indigo_ccd_andor
05:03:20.322792 indigo_server: Driver indigo_ccd_andor can't be loaded (indigo_ccd_andor.so: cannot open shared object file: No such file or directory)
```

I get the error above. That's why I created the `make install` in this pull request. So this works:
```[root@5dac5c4dbb17 ccd_andor]# make install
install -m 0644 /indigo/build/drivers/indigo_ccd_andor.so //usr/local/lib
[root@5dac5c4dbb17 ccd_andor]# cd
[root@5dac5c4dbb17 ~]# indigo_server indigo_ccd_andor
```

Am I missing something to make it work without the necessity of this change?